### ISSUE TITLE
Passing Custom Object as Password does not work

### ISSUE THREAD

**Issue by empower-anil** [plaid/sandbox-custom-users#6]
Hi,
I have tried to pass the Custom Object JSON as a string to the following two api's as password 

1. 'https://sandbox.plaid.com/sandbox/public_token/create'
2. 'https://sandbox.plaid.com/link/item/create'

And receiving the below error message.

'display_message': 'The provided credentials were not correct. Please try again.', 'error_code': 'INVALID_CREDENTIALS', 'error_message': 'the provided credentials were not correct', 'error_type': 'ITEM_ERROR', 'request_id': 'AeEWHDIjKHJHUij', 'suggested_action': None

Appreciate your time in clarifying how to pass the custom object correctly to create custom user programmatically.
Suggestion: It would be nice to have a sandbox API to create a custom user by passing Custom Object as payload 

Thanks
Anil

**Comment by phoenixy1** [comment]
@empower-anil thanks for reporting! Some followup questions:

1) Just to confirm you were using `user_custom` as the username, right?
2) Does the object work if you set it up via the Dashboard at https://dashboard.plaid.com/team/sandbox ? 
3) To help us troubleshoot/reproduce can you provide the exact JSON you were using  as the password?

**Comment by empower-anil** [comment]
> @empower-anil thanks for reporting! Some followup questions:
> 
> 1. Just to confirm you were using `user_custom` as the username, right?
Yes
> 2. Does the object work if you set it up via the Dashboard at https://dashboard.plaid.com/team/sandbox ?
Yes
> 3. To help us troubleshoot/reproduce can you provide the exact JSON you were using  as the password?
PFA attached example JSON. I have passed it as a string in password

[payload.json.zip](https://github.com/plaid/sandbox-custom-users/files/10795909/payload.json.zip)

Thanks,
Anil

**Comment by phoenixy1** [comment]
@empower-anil  Thanks for sending it over! After reviewing, I think there might just be a small syntax error in the JSON you're passing.

I tried the following changes:
1) Delete the comma after the closing bracket on line 153 
2) Delete line 154 (`null` isn't a supported value for` force_error`). 

After making those two changes, I was able to get the configuration object to work when sent via the password field.
